### ISSUE TITLE
inputNumber is not dirty when value changes

### ISSUE THREAD

**Issue by spierala** [primefaces/primeng#10902]
**I'm submitting a ...**
```
[x ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Stackblitz: https://stackblitz.com/edit/primeng-inputnumber-demo-pujvel?file=src/app/app.component.html

**Current behavior**
The primeNg number input does not make the form dirty when changing the value.
The number input itself and the form becomes dirty only after the input loses focus.

**Expected behavior**
When changing the value of the number input, the form status (and the input status) should become dirty immediately. Similar to a regular text input.

**Minimal reproduction of the problem with instructions**
Type a different value in the primeNg number input -> The form does not become dirty.
However, typing in the normal text input -> The form becomes dirty.

**What is the motivation / use case for changing the behavior?**
The number input should behave like other inputs (e.g. a regular text input).

**Please tell us about your environment:**

* **Angular version:** 12.2

* **PrimeNG version:** 12.2

* **Browser:** all? reproduced in Firefox 94, Chrome 96

**Comment by newq** [comment]
Would also be nice if it triggered the forms valueChanges like a regular text input, immediately and not only after blur.

**Comment by denes16** [comment]
It only update the value after blur, on enter keyup, and with up and down arrows of keyboard.

**Comment by mounthorse-slns** [comment]
When this PR would be merged?

**Comment by mertsincan** [comment]
Hi,

So sorry for the delayed response! Improvements have been made to many components recently, both in terms of performance and enhancement. Therefore, this improvement may have been developed in another issue ticket without realizing it. You can check this in the documentation. If there is no improvement on this, can you reopen the issue so we can include it in our roadmap? 
Please don't forget to add your feedback as a comment after reopening the issue. These will be taken into account by us and will contribute to the development of this feature. Thanks a lot for your understanding!

Best Regards,

**Comment by yhoangaleano** [comment]
Sigue aun con problemas
